grakn-test/src/test/java/ai/grakn/test/engine/controller/ImportControllerTest.java

```diff
@@ -137,9 +135,7 @@ private void runAndAssertCorrect(File file, List<String> findByResource, String
     }
 
     private void waitToFinish() {
-        final long initial = new Date().getTime();
-
-        while ((new Date().getTime())-initial < 2*60*60000) {
+        while (true) {
```

Comment:
I did this because the way we had before would timeout. We can switch back to something similar with a greater set timeout.